modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -214,6 +216,55 @@ The `IF NOT EXISTS` and `OR REPLACE` parts of these commands cannot be used toge
 ====
 
 
+==== Create databases using seeding options
+
+Neo4j has built-in support for a seed from a mounted file system (file), FTP server, HTTP/HTTPS server, Amazon S3, Google Cloud Storage, and Azure Cloud Storage.
+To select a seed provider, you need to configure xref:configuration/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[dbms.databases.seed_from_uri_providers].
+
+For more information on seeding from URI, refer to xref::clustering/databases.adoc#cluster-seed-uri[Manage databases in a cluster -> Seed from URI].
+
+.Seed providers supported in Neo4j
+[cols="2,1,2",options="header"]
+|===
+| Seed provider
+| URL scheme
+| URI example
+
+| `FileSeedProvider`
+| `file:`
+| `file:/tmp/backup1.backup`
+
+| `URLConnectionSeedProvider`
+| `ftp:` +
+`http:` +
+`https:`
+| `ftp:://myftp.com/backups/backup1.backup` +
+`/http://myhttp.com/backups/backup1.backup` +
+`/https://myhttp.com/backups/backup1.backup`
```

Comment:
I am not sure about these example URLs. 
Do we need to mention that from 2025.01 URLConnectionSeedProvider does not support `file`. (I saw it on the other page)?

Reply:
What do you mean by 'I'm not sure about these example URLs'? They are not correct? 

We use a slash before 'http' and 'https' because without it they will turn into links.

Reply:
I'll add info that URLConnectionSeedProvider stoped supporting `file`

Reply:
I think we should use _myserver.com_ instead of _myftp.com_ and _myhttp.com_
I am not sure, for example, if the FTP URL looks like this: `ftp:://myftp.com/backups/backup1.backup`. 
I think it should look like this: _ftp://ftp.myserver.com/backups/backup1.backup_ or _ftp://user@ftp.myserver.com/backups/backup1.backup_.
and for the other two:
_https://myhttp.com/backups/backup1.backup_ => _https://myserver.com/backups/backup1.backup_, _http://myhttp.com/backups/backup1.backup_ => _http://myserver.com/backups/backup1.backup_

Reply:
I don't know. I'm not sure about using 'myserver.com'. This is a real web server 

For FTP: FTP URL syntax is described in [RFC](https://en.wikipedia.org/wiki/RFC_(identifier)) [1738](https://datatracker.ietf.org/doc/html/rfc1738), taking the form: ftp://[user[:password]@]host[:port]/[url-path] (the bracketed parts are optional).

This is an interesting finding, and we can improve these example URLs in a separate PR not only here but also on another page.